CHANGELOG.txt

```diff
@@ -1,4 +1,4 @@
-1.2.2 (doi: )
+master
 ===================================
```

Comment:
I know.. this is not a `.rst` but maybe the headline marker could still be length adjusted for cosmetics?

Reply:
Definitely. Done.